src/librawspeed/common/Point.h

```diff
@@ -33,17 +33,14 @@ class iPoint2D {
   iPoint2D operator -= (const iPoint2D& other) { x -= other.x; y -= other.y; return *this;}
   iPoint2D operator - (const iPoint2D& b) const { return iPoint2D(x-b.x,y-b.y); }
   iPoint2D operator + (const iPoint2D& b) const { return iPoint2D(x+b.x,y+b.y); }
-  iPoint2D &operator=(const iPoint2D &b) {
-    x = b.x;
-    y = b.y;
-    return *this;
-  }
+  iPoint2D &operator=(const iPoint2D &b) = default;
   bool operator==(const iPoint2D& rhs){ return this->x==rhs.x && this->y==rhs.y; }
   bool operator!=(const iPoint2D& rhs){ return this->x!=rhs.x || this->y!=rhs.y; }
-	~iPoint2D() {};
-        uint32 area() const { return std::abs(x * y); }
-        bool isThisInside(const iPoint2D &otherPoint) const {
-          return (x <= otherPoint.x && y <= otherPoint.y);
+  ~iPoint2D() = default;
+  ;
```

Comment:
Ugh, i was really sure there was a warning against it / the warning was enabled...
When i get to the rechecking, i will re-check this commit more carefully.
Thanks.